BlueprintUI/Sources/Layout/ConstrainedAspectRatio.swift

```diff
@@ -0,0 +1,77 @@
+/// Constrains the size of the content element to an aspect ratio.
+public struct ConstrainedAspectRatio: Element {
```

Comment:
Non-blocking: Don't love this name but don't have a better suggestion right now.